lobsterpy/cli.py

```diff
@@ -435,6 +442,8 @@ def run(args):
         "automatic-plot-ia",
         "auto-plot-ia",
         "autoplotia",
+        "plot-icohps",
```

Comment:
Not sure this is really the best name as this could be confused with an automatic-plot --integrated.